Simplified/Settings/NYPLSettingsAccountsList.swift

```diff
@@ -254,6 +202,7 @@
     var array = userAddedSecondaryAccounts!
```

Comment:
maybe I missed this in a previous review, but is this forced unwrap safe here?

Reply:
Not sure why it's implemented that way, I assign an empty array to `userAddedSecondaryAccounts` in `init` so it's no longer optional